src/AzureRMTools.ts

```diff
@@ -120,72 +120,56 @@ export class AzureRMTools {
             actionContext.telemetry.properties.isActivationEvent = 'true';
             actionContext.telemetry.properties.fileExt = path.extname(document.fileName);
 
-            let foundDeploymentTemplate = false;
-
-            if (shouldWatchDocument(document)) {
-                if (mightBeDeploymentTemplate(document)) {
-                    // If the documentUri is not in our dictionary of deployment templates, then we
-                    // know that this document was just opened (as opposed to changed/updated).
-                    let stopwatch: Stopwatch;
-                    const documentUri: string = document.uri.toString();
-                    if (!this._deploymentTemplates[documentUri]) {
-                        stopwatch = new Stopwatch();
-                        stopwatch.start();
-                    }
+            const stopwatch = new Stopwatch();
+            stopwatch.start();
```

Comment:
Does this need to be accompanied by a stop?

Reply:
No, it's just a thin wrapper on Date.now(), but thanks for asking.